setup.cfg

```
@@ -20,7 +20,7 @@ install_requires =
     astropy>=4.0
     glue-core>=1.6.1
     regions>=0.4
-    specutils>=0.7
+    specutils>=2.0.0rc0
```

```suggestion
    specutils>=1.20
```
should set this so the py310 tests can continue to be run, and leave 2.0 for the -dev tests for now.